x-pack/plugins/security_solution/public/common/components/exceptions/builder/index.tsx

```diff
@@ -43,7 +43,7 @@ interface OnChangeProps {
 }
 
 interface ExceptionBuilderProps {
-  exceptionListItems: ExceptionListItemSchema[];
+  exceptionListItems: ExceptionsBuilderExceptionItem[];
```

Comment:
I think this should still just be `ExceptionListItemSchema`. The builder should only either take 1) no exception list items (if creating new one), or 2) existing exception list items to edit. The `ExceptionsBuilderExceptionItem` was meant to represent an internal kind of in-between state that exception list items can be in while in the builder, but they get passed back out as `ExceptionListItemSchema | CreateExceptionListItemSchema`.

Reply:
UPDATE: Chatted with @peluja1012 offline. Leaving this as is. I understand this was changed to allow for the 'preconfigured' exception items for endpoint that are added by us when user is first creating an exception.